pkg/engine/utils/exceptions.go

```diff
@@ -40,6 +43,14 @@ func MatchesException(
 					return nil
 				}
 			}
+			image := imageInfo.String()
+			if polex.Spec.VerifyImages.ImageReferences != nil {
+				for _, imageRef := range polex.Spec.VerifyImages.ImageReferences {
+					if !wildcard.Match(imageRef, image) {
+						return nil
+					}
+				}
+			}
```

Comment:
You can call [matchReferences()](https://github.com/kyverno/kyverno/blob/main/pkg/engine/internal/imageverifier.go#L93-L100) instead. If it returns false, then there are no exceptions matched.